falcon/cmd/print_routes.py

```diff
@@ -54,8 +54,8 @@ def traverse(roots, parent='', verbose=False):
                     if func.__name__ != 'method_not_allowed':
                         print('-->{0} {1}:{2}'.format(
                             method,
-                            inspect.getsourcefile(func),
-                            inspect.getsourcelines(func)[1]))
+                            inspect.getsourcefile(func.func),
```

Comment:
[fix] This will break for regular responder objects, since they don't have a `func` attribute. This didn't show up in the tests because `test_cmd_print_api` is rather basic and doesn't actually test an API that had been configured with routes to actual resources. You'll need to add that test and then fix the issue here that it exposes.